docs/source/installation.rst

```diff
@@ -0,0 +1,26 @@
+Installation
+============
+
+scikit-COSMO is, by design, relatively lightweight, and only requires that `numpy <numpy.org>`_
+and the most recent release of `scikit-learn <scikit-learn.org>`_ be installed.
+
+Install with pip
```

Comment:
I would remove this installation method until we have a release installable directly from PyPI

Reply:
Sure! This is the _original_ documentation on installation, however, so I figured to include it from that standpoint.

Reply:
Yes, but this method is now replaced by the section below `Install from Source`

Reply:
I see no need to keep it -- updated!